Makefile

```diff
@@ -12,22 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GO ?= go
-EPOCH_TEST_COMMIT ?= f9e02affccd51702191e5312665a16045ffef8ab
+GO := go
+EPOCH_TEST_COMMIT := f9e02affccd51702191e5312665a16045ffef8ab
 PROJECT := github.com/kubernetes-incubator/cri-containerd
-BINDIR ?= ${DESTDIR}/usr/local/bin
-BUILD_DIR ?= _output
+BINDIR := ${DESTDIR}/usr/local/bin
+BUILD_DIR := _output
 # VERSION is derived from the current tag for HEAD plus amends. Version is used
 # to set/overide the criContainerdVersion variable in the verison package for
 # cri-containerd.
 VERSION := $(shell git describe --tags --dirty)
 # strip the first char of the tag if it's a `v`
 VERSION := $(VERSION:v%=%)
-TARBALL ?= cri-containerd-$(VERSION).tar.gz
-ifdef BUILD_TAGS
-BUILD_TAGS := -tags $(BUILD_TAGS)
-endif
-GO_LDFLAGS := -ldflags '-X $(PROJECT)/pkg/version.criContainerdVersion=$(VERSION)'
+TARBALL := cri-containerd-$(VERSION).tar.gz
+BUILD_TAGS := apparmor
```

Comment:
a little nervous forcing the apparmor build tag.. let's talk about this on our call.

Reply:
I think the default is mainly targeting at Kubernetes upstream test environment. Now we are mainly testing on Ubuntu and COS, and both support apparmor.